pkg/diagnose/connectivity/trace_request.go

```diff
@@ -32,13 +32,16 @@ func newHTTPClient() *http.Client {
 	}
 }
 
-func RunDatadogConnectivityChecks() error {
-
+// RunDatadogConnectivityDiagnose send requests to all known endpoints for all domains
```

Comment:
`all known endpoints`: This is not true but you can use something like `send requests to endpoints` for example.

Reply:
I applied your suggestion 👍